app/views/monthlyreturns/SubmissionUnsuccessfulView.scala.html

```diff
@@ -0,0 +1,77 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        layout: templates.Layout,
+        govukButton: GovukButton,
+        formHelper: FormWithCSRF,
+        h1: components.H1,
+        h2: components.H2,
+        paragraph: components.Paragraph,
+        bulletList: components.BulletList,
+        link: components.Link,
+        govukWarningText: GovukWarningText,
+        govukDetails: GovukDetails,
+        config: play.api.Configuration
```

Comment:
We should use **FrontendAppConfig**. No need to use application.conf directly.

Refere existing code

https://github.com/hmrc/cis-frontend/blob/main/app/forms/mappings/MonthYearDateFormatter.scala#L38

https://github.com/hmrc/cis-frontend/blob/main/app/forms/mappings/MonthYearDateFormatter.scala#L73